.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+// Provide a blank config so that build doesn't fail due to ESLint warnings from react-scripts build step
+// The plugins need to be loaded to avoid getting errors from eslint-disable comments and eslint
+// npm lint script uses an alternative configuration provided in package.json
+
+module.exports = {
```

Comment:
Does adding this make sense @calebmshafer ? I'm skeptical this may be losing more than desired concerning eslint rules. I'm not sure if this work-around makes sense.

Reply:
@calebmshafer @BaoTon still concerned about this?

Reply:
I'm still concerned - seems like everyone who is using @bentley/react-scripts are running into the same issue.

I'll merge this the way it is if we don't get anything by tomorrow since this is blocking our releases currently.

Reply:
Sorry, haven’t had a chance yet today. I’ll take a look at it in a couple hours.

Reply:
You can immediately see the effect @bentley/react-scripts 3.4.9 has if you remove the .eslintrc.js file and run "npm run lint" on this branch.

Reply:
That seems to work from me but I did get a different error when running `npm start`.  Any idea?

```
generated: src/common/Geometry/SampleCurveFactory.d.ts
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '@bentley/monaco-editor/monaco-webpack-plugin'
Require stack:
- /Users/calebshafer/dev/github/frontend-sample-showcase/config-overrides.js
- /Users/calebshafer/dev/github/frontend-sample-showcase/node_modules/react-app-rewired/config-overrides.js
- /Users/calebshafer/dev/github/frontend-sample-showcase/node_modules/react-app-rewired/scripts/start.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/Users/calebshafer/dev/github/frontend-sample-showcase/config-overrides.js:6:29)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/calebshafer/dev/github/frontend-sample-showcase/config-overrides.js',
    '/Users/calebshafer/dev/github/frontend-sample-showcase/node_modules/react-app-rewired/config-overrides.js',
    '/Users/calebshafer/dev/github/frontend-sample-showcase/node_modules/react-app-rewired/scripts/start.js'
  ]
}
```

Reply:
Ah looks like I updated the monaco-editor version and it broke something.  @StefanRetief mind taking a look?

For this PR though, I'll just lock it down to the previous version.

Reply:
@BaoTon can you give this a try now?

Reply:
@calebmshafer PR #158 will fix a lot of the editor issues, it's just been merged in

Reply:
Getting linting from ViewCreator2d.ts? It doesn't look relevant.

Reply:
Why does that even exist any more? @roopksaini moved the API to imodeljs-frontend.

Reply:
@BaoTon are you able to reproduce that locally.   I tried when I saw the build fail but wasn't able to.

Reply:
Yeah it happens every time I run it locally.

Reply:
Hm okay, let me try on my windows machine.  Was only running MacOS before.

Reply:
Okay, I can reproduce on Windows.  Digging now.

Reply:
FWIW I deleted the two ViewCreator2d classes and switched to using the one from imodeljs-frontend.

Reply:
@calebmshafer Any luck with the windows linting?

Reply:
Didn't get a chance to look yesterday but think I have a fix.  It looks like the config was messed up in the package.json.  I'll have a fix in a few minutes.

Reply:
Should be good to go now.  Let me know if you continue to have issues.

Reply:
Awesome, thanks Caleb.